src/content/plugins/eval-source-map-dev-tool-plugin.md

```diff
@@ -35,11 +35,26 @@ The following options are supported:
 
 T> Setting `module` and/or `columns` to `false` will yield less accurate source maps but will also improve compilation performance significantly.
 
+T> When using this plugin in [development mode](/concepts/mode/#mode-development), you must manually set `devtool: false`.
 
 ## Examples
 
 The following examples demonstrate some common use cases for this plugin.
 
+### Basic Use Case
+
+To write out `devtool: eval-source-map` using the plugin you can use the following configuration:
```

Comment:
Can you rephrase this? I would expect this to note that in development it is necessary to set `devtool: false` in order for custom configured plugin instance to work, default use case (`devtool: 'eval-source-map'`) as mentioned above should work just fine for both development and production modes.